### ISSUE TITLE
Core: Chain sponsorship is no longer working from the browser

### ISSUE THREAD

**Issue by djlmills** [interbit/interbit#361]
The PPC model used by the accounts chain, template and other interbit platform applications are dependent on private user chains created using chain sponsorship from the browser. This process has been working for several months up with `interbit-core <= 0.8.2`, but no longer works with `interbit-core 0.11.0`.

The error is: `Error: waitForState timeout after waiting 20000 ms`

The interbit platform sponsorship code running in `redux-saga` on the browser is:

```js
function* sponsorChain({
  interbit,
  cli,
  publicKey,
  publicChainAlias,
  chainAlias
}) {
  const { blockMaster, sponsorChainId, covenantHash } = yield select(
    selectors.getSponsorConfig,
    { publicChainAlias, privateChainAlias: chainAlias }
  )
  if (!blockMaster || !sponsorChainId || !covenantHash) {
    throw new Error(
      'public chain does not contain expected chain sponsorship configuration'
    )
  }

  const genesisConfig = interbit.createDefaultSponsoredChainConfig({
    blockMaster,
    myPublicKey: publicKey,
    sponsorChainId
  })

  const genesisBlock = interbit.createGenesisBlock({
    config: genesisConfig,
    configChanges: { covenantHash }
  })

  const chainId = genesisBlock.blockHash

  yield call(cli.sendChainToSponsor, {
    parentChainId: sponsorChainId,
    publicKeys: [publicKey, blockMaster],
    genesisBlock
  })
  return chainId
}
```

**Comment by djlmills** [comment]
This requires a new core version.

**Comment by pekeler** [comment]
Can we close this with 0.13.0?

**Comment by djlmills** [comment]
Resolved with `interbit-core` 0.13.0
